### PR TITLE
Prod 907 - Same code from GItlab

### PR DESCRIPTION
**Pull request by Kaltner**

### CODE REVIEW

File: src/commands/run.ts [hunk 1]
```diff
@@ -616,35 +632,120 @@ export default class Run extends Command {
...
+              headers: {
+                Authorization: this.accessToken,
+              },
+            },
+          )
+
+          value = data
+        } catch (err) {
+          this.debug('Error %0', err)
+          value = ''
```
Comment (maumercado) [inline src/commands/run.ts:730]: don't need this since that it's the value by default :)

File: src/commands/run.ts [hunk 2]
```diff
@@ -616,35 +632,120 @@ export default class Run extends Command {
...
+        }
+        parsedEnv.push(`${envKey}=${value}`)
+        console.log('Parsed ENVS', parsedEnv)
```
Comment (maumercado) [inline src/commands/run.ts:733]: forgotten console.log?

**Review by maumercado — CHANGES_REQUESTED** [review]
awesome job... couple of nits

File: src/commands/run.ts (hunk 2 above)
Comment (Kaltner) [reply src/commands/run.ts]: Removed! Thanks

File: src/commands/run.ts (hunk 1 above)
Comment (Kaltner) [reply src/commands/run.ts]: Removed! Thanks

**Review by maumercado — APPROVED** [review]
LGTM Awesome Job!

**Pull request merged**